csrankings.csv

```diff
@@ -35,7 +35,6 @@ Aamir Cheema,Monash University,http://www.aamircheema.com,chWbG70AAAAJ
 Aanjhan Ranganathan,Northeastern University,https://www.aanjhan.com,Jid_EAkAAAAJ
 Aaram Yun,UNIST,http://aaramyun.unist.ac.kr,NOSCHOLARPAGE
 Aaron Bernstein,Rutgers University,http://bernsteinaaron.com,NOSCHOLARPAGE
-Aaron Bloomfield,University of Virginia,https://engineering.virginia.edu/faculty/aaron-bloomfield,NOSCHOLARPAGE

```

Comment:
Please explain / justify this removal. He still appears here: https://engineering.virginia.edu/departments/computer-science/faculty.